packages/web-components/fast-foundation/src/menu-item/menu-item.ts

```diff
@@ -221,18 +222,33 @@ export class FASTMenuItem extends FASTElement {
                 this.expandAndFocus();
                 return false;
 
+            case keyEscape:
+                // close submenu
+                if (this.expanded) {
+                    this.closeSubMenu();
+                    return false;
+                }
+
```

Comment:
This case needs a `break` after the `if` statement

Reply:
Done!